packages/backend/src/models/UserModel.ts

```diff
@@ -559,7 +569,10 @@ export class UserModel {
                 'users.user_id',
             )
             .where('token_hash', tokenHash)
-            .select<(DbUserDetails & DbPersonalAccessToken)[]>('*');
+            .select<(DbUserDetails & DbPersonalAccessToken)[]>(
```

Comment:
don't you need to include `& {organization_created_at: Date}` on this select ?

Reply:
I updated `DbUserDetails` to have the `organization_created_at`